### PR TITLE
fix: unit test and linter not running for aries-agentd

### LINKED ISSUE #736: unit test and linter not running for aries-agentd

### PR DESCRIPTION
**Pull request by fqutishat**
closes #736

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: scripts/check_lint.sh [hunk 1]
```diff
@@ -16,3 +16,4 @@ if [ ! $(command -v ${DOCKER_CMD}) ]; then
 fi
 
 ${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/workspace -w /opt/workspace golangci/golangci-lint:v1.21 golangci-lint run
+${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/tmp -w /opt/tmp golangci/golangci-lint:v1.21  /bin/bash -c "mkdir -p /opt/workspace;cp -r /opt/tmp/. /opt/workspace;cd /opt/workspace/cmd/aries-agentd;cp ../../.golangci.yml .;golangci-lint run"
```
Comment (troyronda) [inline scripts/check_lint.sh]: Why not just cd into cmd/aries-agentd? Why copy?

File: scripts/check_unit.sh [hunk 2]
```diff
@@ -8,10 +8,28 @@ set -e
...
 
-# Packages to exclude
+go generate ./...
+pwd=`pwd`
+echo "" > coverage.txt
+
+# Running aries-framework-go unit test
 PKGS=`go list github.com/hyperledger/aries-framework-go/... 2> /dev/null | \
                                                  grep -v /mocks`
+go test $PKGS -count=1 -race -coverprofile=profile.out -covermode=atomic -timeout=10m
```
Comment (troyronda) [inline scripts/check_unit.sh]: Suggest making a bash function to get rid of the copy & paste.

File: scripts/check_lint.sh (hunk 1 above)
Comment (fqutishat) [reply scripts/check_lint.sh]: I need .golangci.yml  inside cmd/aries-agentd

Comment (fqutishat) [reply scripts/check_lint.sh]: done

File: scripts/check_unit.sh (hunk 2 above)
Comment (fqutishat) [reply scripts/check_unit.sh]: done

File: scripts/check_lint.sh [hunk 3]
```diff
@@ -16,3 +16,4 @@ if [ ! $(command -v ${DOCKER_CMD}) ]; then
 fi
 
 ${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/workspace -w /opt/workspace golangci/golangci-lint:v1.21 golangci-lint run
+${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/workspace -w /opt/workspace/cmd/aries-agentd golangci/golangci-lint:v1.21 golangci-lint run -c ../../.golangci.yml
```
Comment (troyronda) [inline scripts/check_lint.sh]: Consider making the linter version a variable above. We will probably forgot to update each line :).

**Pull request merged**